### PR TITLE
Adding default storage class for oVirt platform

### PR DESCRIPTION
**Pull request by rgolangh**
Enhancment: https://github.com/openshift/enhancements/pull/61

WIP:
[] the enhancement document needs an update
[] the CSI driver needs to be published under openshift

### CODE REVIEW

File: assets/ovirt.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: thin
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/ovirt-csi
```
Comment (jsafrane) [inline assets/ovirt.yaml]: `kubernetes.io` prefixes are reserved for Kubernetes and we expect that they're in-tree. You should use something else.

**Comment by cuppett** [comment]
@jsafrane @gnufied Can we get this one reviewed and merged (enables higher level RHV PRs to begin to merge enabling the platform)?

File: assets/ovirt.yaml (hunk 1 above)
Comment (gnufied) [reply assets/ovirt.yaml]: Just to be sure, this is the name of the CSI driver too right? Provisioner requires that driver name should be same as provisioner name here.

**Comment by jsafrane** [comment]
i talked to @rgolangh the other day, the PR is OK, however, it has little sense without an actual CSI driver. This PR should be the last step of CSI driver installation.

**Pull request closed without merging**